README.md

```diff
@@ -56,7 +56,7 @@ route('events.venues.show', [event, venue]) // Returns '/events/1/venues/2'
 
 Practical AJAX example:
 
-```javascript
+```js
```

Comment:
Any reason for this line? The rest seem good, just curious what value this provides. Thanks!